skyportal/handlers/api/allocation.py

```diff
@@ -418,6 +418,26 @@ def get(self, allocation_id=None):
 
             allocations = session.scalars(allocations).unique().all()
             # order by allocation.instrument.telescope.name, then instrument.name, then pi
+
+            apiImplements = self.get_query_argument('apiImplements', None)
```

Comment:
Could we just enforce a set of valid values here?

Reply:
Done.